src/Rel8/Expr/Aggregate.hs

```diff
@@ -132,6 +133,16 @@ sum = unsafeMakeAggregate toPrimExpr (castExpr . fromPrimExpr) $
     }
 
 
+-- | Corresponds to @avg@. The same caveats that apply to 'sum' apply here.
```

Comment:
> The same caveats apply

Do they? Given `x1, x2, ..., x3` isn't the average bounded by `min(x1, x2, ..., x3)` and `max(x1, x2, ..., x3)`. That is - can it overflow? Maybe there's something funky with the casts though, but it seems safe to me

Reply:
Okay, I'll expand that comment to clarify. You're right that overflow isn't a problem.